analyzer/server.go

```diff
@@ -176,8 +176,9 @@ func (s *Server) initialize() (err error) {
 
 	api.RegisterTopologyAPI(s.HTTPServer, tr)
 
-	api.RegisterPacketInjectorAPI(piClient, s.TopologyServer.Graph, s.HTTPServer)
-
+	if config.GetConfig().GetBool("analyzer.inject_enabled") {
```

Comment:
I propose to add a function in order to return if the resource accepts write operation : https://github.com/skydive-project/skydive/blob/master/api/handler.go#L46, (ex: IsEnabled). So that in the Create/Delete functions we check whether the handler is allowed to modify the resource or not. If not we could return http.StatusForbidden.